### PR TITLE
dark theme using provider and shared_pref package

### PR DESCRIPTION
**Pull request by rabin245**
fixed issue where reopening the app would initialize the theme to dark as loading the shared preference took some time.

### CODE REVIEW

File: lib/shared_pref/theme_shared_preference.dart [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import 'package:shared_preferences/shared_preferences.dart';
+
+class ThemeSharedPreference {
+  static loadTheme() async {
+    SharedPreferences sharedPreference = await SharedPreferences.getInstance();
+    bool darkPref = sharedPreference.getBool('isDark') ?? true;
+    return darkPref;
+  }
+
+  setTheme(bool value) async {
```
Comment (anwesh-b) [inline lib/shared_pref/theme_shared_preference.dart:10]: static?

**Comment by anwesh-b** [comment]
Lets update version as well :)

File: lib/shared_pref/theme_shared_preference.dart (hunk 1 above)
Comment (rabin245) [reply lib/shared_pref/theme_shared_preference.dart:10]: tyo object banairakhna paryo ani code pani clutter vayo tei vaera function lai static banako. suruma theme load garda ekchoti matra run hune vayeko le

File: android/app/local.properties [hunk 2]
```diff
@@ -0,0 +1,8 @@
+## This file must *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
+#
+# Location of the SDK. This is only used by Gradle.
+# For customization when using a Version Control System, please read the
+# header note.
+#Mon Dec 27 21:19:35 NPT 2021
+sdk.dir=C\:\\Users\\Zaxiya\\AppData\\Local\\Android\\Sdk
```
Comment (anwesh-b) [inline android/app/local.properties:8]: .

Comment (rabin245) [reply android/app/local.properties:8]: how to revert commit in remote xD

**Pull request merged**